.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+  schedule:
+    # at 12:00 on Sunday (UTC)
+    - cron: '0 12 * * 0'
```

Comment:
Forgot to ask, but is this time ok? The CI will automatically run at this time every week (for at most 3 months if there are no changes to the repository, I think).